packages/dara-components/js/common/select/select.tsx

```diff
@@ -53,6 +50,11 @@ function hasListSection(items: Item[] | ListSection[]): items is ListSection[] {
 function isStringArray(value: any): value is string[] {
     return Array.isArray(value) && value.every((item) => typeof item === 'string');
 }
+
+/** Type guard to check if an object is an Item */
+function isItem(obj: any): obj is Item {
+    return Object.prototype.hasOwnProperty.call(obj, 'value');
```

Comment:
Could also add `typeof obj === 'object'`, otherwise might call `value` on null and throw an error

Reply:
good point, fixed

Reply:
Actually, sorry I blanked on it when commenting - `typeof null` resolves to `object` as well (because JavaScript) so you might need to explicitly check for null